pkg/utils/skip_rules.go

```diff
@@ -35,15 +35,17 @@ const (
 )
 
 var (
-	skipRulesPattern               = regexp.MustCompile(`(#ts:skip=[ \t]*(([A-Za-z0-9]+[.-]{1}){3,5}([\d]+)){1}([ \t]+.*){0,1})`)
+	ruleNameRegex                  = `((([ A-Za-z0-9]+[.-]{1})){2,5}([\d]+)){1}`
+	ruleNamePattern                = regexp.MustCompile(ruleNameRegex)
+	skipRulesPattern               = regexp.MustCompile(`(#ts:skip=` + ruleNameRegex + `([ \t]+.*){0,1})`)
 	skipRulesPrefix                = "#ts:skip="
 	infileInstructionNotPresentLog = "%s not present for resource: %s"
 )
 
 // GetSkipRules returns a list of rules to be skipped. The rules to be skipped
 // can be set in terraform resource config with the following pattern:
 // #ts:skip=AWS.S3Bucket.DS.High.1043
-// $ts:skip=AWS.S3Bucket.DS.High.1044 reason to skip the rule
+// #ts:skip=AWS.S3Bucket.DS.High.1044 reason to skip the rule
```

Comment:
does this change has any relevance or significance?

Reply:
For relevance to the issue, I think no. But for significance, if we read the comment on line 46 it might create confusion for the reader that we accept # and $ both for skipping rules.

Reply:
okay then